### DISCUSSION TITLE
Configuration Cluster server

### DISCUSSION THREAD

**Discussion by massimoguida** [pimcore/pimcore#16298] [Q&A / Support]
Hi,
I'm new to using Pimcore.  I'm looking at how to setup pimcore to run on a server cluster to do HA. I found this Page without but It Is empty https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Setup_and_Hosting/Cluster_Setup/.
Could you help me?

**Comment by dpfaffenbauer** [comment]
you can run pimcore with kubernetes, example see here: https://pimcore.com/en/resources/blog/kubernetes-with-pimcore_a271065

**Reply by massimoguida** [reply]
Thanks a lot. 
I will try.
Regards

**Reply by talsan74** [reply]
the above link isn't land to correct path
